src/components/Dashboard/CardRenderer.jsx

```diff
@@ -71,6 +72,7 @@ const CardRenderer = React.memo(
     onFetchData, // eslint-disable-line
     onSetupCard, // eslint-disable-line
     timeGrain, // eslint-disable-line
+    onClick, //eslint-disable-line
```

Comment:
I think we should establish the pattern of setting a noop as a default for optional function props so we can safely call these functions without having to check if they're `null`.

So in the defaults it's just
```
onClick: () => {}
```

Reply:
default noop function prop on the `CustomCard` component

Reply:
Do we need to also declare defaults within this component to remove the eslint disable?